desktop-cosmic/cosmic-app-library/spec

```diff
@@ -0,0 +1,4 @@
+VER=1.0.0-alpha.6
```

Comment:
You should be using ~ to indicate pre-release:

```
UPSTREAM_VER=1.0.6-alpha.6
VER=${VER/\-/~}
```

And use `$UPSTREAM_VER` below for `SRCS=`.

Reply:
Same for all below.